src/Utils/SeoMetadataResolver.php

```diff
@@ -42,11 +42,25 @@ public function __construct(Title $title, Excerpt $excerpt, array $metadataConfi
      *
      * @param object|null $entity
      *
+     * @param bool $includePrefix
+     * @param bool $includeSuffix
      * @return string
      */
-    public function metaTitle(?object $entity): string
+    public function metaTitle(?object $entity, bool $includePrefix, bool $includeSuffix): string
```

Comment:
Please set default values to `$includePrefix` and `$includeSuffix`, otherwise this would induce a BC break.